### PR TITLE
tetragon: Add pprof http support

### PR DESCRIPTION
**Pull request by anjmao**
This PR adds support for pprof http endpoints.

Fixes: https://github.com/cilium/tetragon/issues/550

Signed-off-by: Andzej Maciusovic [andzej.maciusovic@gmail.com](mailto:andzej.maciusovic@gmail.com)

### CODE REVIEW

File: cmd/tetragon/main.go [hunk 1]
```diff
@@ -17,6 +19,7 @@ import (
 	"time"
 
 	"github.com/cilium/tetragon/api/v1/tetragon"
+
```
Comment (kkourt) [inline cmd/tetragon/main.go:22]: nit: is there a reason for this added space?

**Review by kkourt — APPROVED** [review]
LGTM, Thanks!

**Comment by anjmao** [comment]
@kkourt Rebased and added `Signed-off-by` text.

**Comment by kkourt** [comment]
> @kkourt Rebased and added `Signed-off-by` text.

Could you also please add it in the git commit as well? `git commit --amend --signoff` should do the trick.
Thanks!

**Comment by tixxdz** [comment]
Check patch still complains. Also it would have been great if you added ina  separate patch some docs under `docs/debugging/README.md` a section for profiling Tetragon agent with examples and output, so we can run them.

Thank you, LGTM!

**Comment by anjmao** [comment]
@kkourt Ah, sorry about that. Added signoff to git commit.

**Review by kkourt — APPROVED** [review]

**Comment by kkourt** [comment]
> @kkourt Ah, sorry about that. Added signoff to git commit.

Great! Many thanks!

**Pull request merged**